secp384r1/src/lib.rs

```diff
@@ -12,10 +12,12 @@
 //! Source: <https://neuromancer.sk/std/nist/P-384>
 //!
 //! Curve information:
-//! * Base field: q = TODO
-//! * Scalar field: r = TODO
-//! * a = TODO
-//! * b = TODO
+//! * Base field: q =
+//!   0xfffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffeffffffff0000000000000000ffffffff
+//! * Scalar field: r =
+//!   0xffffffffffffffffffffffffffffffffffffffffffffffffc7634d81f4372ddf581a0db248b0a77aecec196accc52973
+//! * a = 0xfffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffeffffffff0000000000000000fffffffc
```

Comment:
It seems like this is -3, right? Let's use that.
```suggestion
//! * a = -3
```

Reply:
That makes a lot of sense :)